cypress/e2e/article.cy.js

```diff
@@ -1,15 +1,61 @@
+import faker from 'faker';
+
 describe('', () => {
```

Comment:
The describe block should have a description. This help in identifying what the test suite is for.

Reply:
done

Reply:
describe('**test suite description**', () => ....